include/tvm/meta_schedule/database.h

```diff
@@ -357,6 +357,18 @@ class Database : public runtime::ObjectRef {
    */
   TVM_DLL static Database JSONDatabase(String path_workload, String path_tuning_record,
                                        bool allow_missing);
+  /*!
+   * \brief Create a database merged from multiple databases.
+   * \param preferred The preferred database. If the preferred database responses to a query,
+   * all other databases will be ignored.
+   * \param databases The databases to be merged.
+   * \param fallback The fallback databases. If all the databases didn't answer a query,
+   * the fallback databases will be used.
```

Comment:
Is there any inclusion hierarchy among the three inputs? From the current C++ documents, my understanding is that `databases` must contains all databases in `preferred` and `fallback`. But the documents and example on Python side obviously doesn’t prove my understanding correct. Would it be a good idea to tell a bit about their relation?

Reply:
There is no hierarchy. There is no requirement that `databases` must contain `preferred` or `fallback`. There is no relationship between those three fields. Would be great if you could suggest an enhancement to the doc to avoid confusion :-)

Reply:
Sorry I guess I now understand the logic - we won’t merge databases in either `preferred` or `fallback`. Thank you! Is it okay to add a short note saying that `preferred` and `fallback` won’t be merged?